### PR TITLE
Cap tokens

### PR DESCRIPTION
**Pull request by NimRegev**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-tokens/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-tokens/docs/

File: _docs/administration/oauth-pat-setup.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+data:
+  appId: # application ID of your OAuth app from GitHub
+  clientId: # client ID of your OAuth app from GitHub
+  clientSecret: # client secret of your OAuth app from GitHub
+  url: https://github.com
+```
+2. Apply the secret to the runtime cluster:  
+   `kubectl apply -f <filename>`   
+   where:   
+   `<filename>` is the `metadata.name`, `github-oauth2` in the example.
```
Comment (andrii-codefresh) [inline _docs/administration/oauth-pat-setup.md:53]: the filename can be any. all data is inside

File: _docs/administration/oauth-pat-setup.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+---
+
+Administrators can create, register and authorize CSDP as a OAuth2 App in GitHub. Users can then authorize access to GitHub instead of generating a personal access token from their Git providers.
+
+To set up OAuth2 authorization for GitHub, as an admin you must create:
+* GitHub OAuth2 Application
+* Secret in runtime cluster with OAuth2 Application credentials
+* Config-map that references the secret
+
+> An OAuth2 Application must be created for every runtime in your deployment.
```
Comment (andrii-codefresh) [inline _docs/administration/oauth-pat-setup.md:15]: Administrator must have config-map and secret in each runtime. But one application can be reused for many runtimes (the same credentials can be used in many secrets)

File: _docs/administration/user-settings.md [hunk 3]
```diff
@@ -1,22 +1,75 @@
...
+   lightbox="true" 
+   file="/images/administration/user-settings/oauth-user-authentication.png" 
+   url="/images/administration/user-settings/oauth-user-authentication.png" 
+   alt="Authorizing access with OAuth2" 
+   caption="Authorizing access with OAuth2"
+   max-width="30%" 
+   %}
+  * Complete the verification, if you have two-factor authentication for example.
+  The token is generated, and you are automatically directed to the Git Personal Access Tokens page. 
+ 1. For Git personal access tokens: Paste the generated token in the **Token** field, and select **+Add Token**.
```
Comment (andrii-codefresh) [inline _docs/administration/user-settings.md:60]: Wrong numbered list structure for options `OAuth2` and `personal access token`

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-tokens/docs/

File: _docs/administration/user-settings.md (hunk 3 above)
Comment (NimRegev) [reply _docs/administration/user-settings.md]: I changed the structure but known issue with lists in md

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-tokens/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-tokens/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-tokens/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-tokens/docs/

**Pull request merged**